packages/fonts/package.json

```diff
@@ -31,7 +31,10 @@
      "./fonts.json": {
          "import": "./dist-module/fonts.json",
          "require": "./dist/fonts.json"
-     }
+     },
+     ".": {
+      "default": "./src/index.js"
+    }
```

Comment:
Note to self: we need to fix this for webpack somehow.

This should point to `dist/index.js`, just like the other entries also point to `dist`/`dist-module`.

We just need to tell webpack that it should still look in the `src` folder in general.